Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -91,13 +91,15 @@ final class ConversationOptionsViewModel {
         if configuration.allowGuests && configuration.isCodeEnabled {
             fetchLink()
         }
+
     }
 
     private func updateRows() {
-        state.rows = computeVisibleRows()
+        state.rows = computeVisibleRowsForGuests()
     }
 
-    private func computeVisibleRows() -> [CellConfiguration] {/// TODO: copy?
+    private func computeVisibleRowsForGuests() -> [CellConfiguration] {/// TODO: copy?
```

Comment:
The name can stay the same, since this view model is only about guests.

```suggestion
    private func computeVisibleRows() -> [CellConfiguration] {/// TODO: copy?
```